CollAction/Frontend/app/global/RichTextRequiredValidation.ts

```diff
@@ -0,0 +1,25 @@
+if (typeof $ !== "undefined" && typeof $.validator !== "undefined" && typeof $.validator.unobtrusive !== "undefined") {
```

Comment:
Wouldn't a better place for this be inside the rich-text editor component, in the render function?

Reply:
This validation function is automatically added to the <input type="hidden"> element that is bound to a ViewModel property with a [RichTextRequired] attribute. It's not part of the rich text editor, but of jquery validation.